src/DSP/CombFilter.cpp

```diff
@@ -57,19 +59,47 @@ void CombFilter::filterout(float *smp)
     memcpy(&input[mem_size-buffersize], smp, buffersize*sizeof(float));
     for (int i = 0; i < buffersize; i ++)
     {
-        // calculate the feedback sample positions in the buffer
-        float pos = float(mem_size-buffersize+i)-delay;
-        // add the fwd and bwd feedback samples to current sample
-        smp[i] = smp[i]*gain + tanhX(
-            gainfwd * sampleLerp( input, pos) - 
-            gainbwd * sampleLerp(output, pos)); 
-        // copy new sample to output buffer
-        output[mem_size-buffersize+i] = smp[i];
+        if (reversed)
```

Comment:
in this conditional branch is the reverse processing with the strange harsh sound added to the audio input after some time